.github/workflows/ci.yaml

```diff
@@ -72,6 +72,10 @@ jobs:
             twisted-version: "trunk"
             experimental: true
 
+          - python-version: "3.9"
+            twisted-version: "trunk"
+            experimental: true
```

Comment:
Why is this experimental ?

I see that the build is green.

Twisted trunk supports 3.9 on Linux, Windows and macOS.

I did the initial review, but maybe `experimental` should be renamed to `allow_to_fail` or 'ok_to_fail` :)

Also, I see that 3.8 with Twisted trunk is also green .

Reply:
It's experimental because it's using Twisted trunk, rather than the latest release. This functions to catch regressions or compatibility issues with Twisted trunk (note that `treq.testing` uses `twisted.test.iosim` which is nominally not a public API). There are non-experimental builds for CPython 3.8 and 3.9 with the latest Twisted release.